src/deckgl-layers/sharedstreets-layer/sharedstreets-layer.js

```diff
@@ -116,13 +156,69 @@ export default class SharedstreetsLayer extends CompositeLayer {
     return [
       new DeckGLTileLayer({
         getTileData: this.getTileData.bind(this),
-        maxZoom: 12,
-        minZoom: 12,
-        renderSubLayers: this.renderSubLayers.bind(this)
+        maxZoom: 13,
+        minZoom: 10,
+        renderSubLayers: this.renderSubLayers.bind(this),
+        onDataLoaded: this.onDataLoaded.bind(this)
       })
     ];
   }
 
+  /**
+   * return domain for the given tile  
+   * @param {*} layer 
+   * @param {*} tileId 
+   * @param {*} fieldName 
+   * @param {*} channel name of visual encoding channel for this field
+   * @param {*} data data that's loaded in the tile
+   */
+  getTileDomain(layer, tileId, fieldName, channel, data) {
+    const {id: layerId} = layer;
+    const keyObj = {
+      layerId,
+      fieldName,
+      tileId
+    };
+    const key = JSON.stringify(keyObj, Object.keys(keyObj).sort());
```

Comment:
Make the filter array a constant

Reply:
Filter array? I didn't use a filter array here. Can you say more about it?

Reply:
The second argument (`Object.keys(keyObj).sort()`) is a constant. Don't recalculate it every time.